### ISSUE TITLE
Readme instructions are out dated

### ISSUE THREAD

**Issue by dxk3355** [OneDrive/onedrive-sample-apibrowser-dotnet#17]
The read me says to create a 'Mobile' type application.  The only options available on the app registration page are Web, Web API, and Native Application.

**Comment by sven5** [comment]
Yes please update readme and code to reflect current APIs and frameworks.
